### DISCUSSION TITLE
`Link` only allows 1 child (Next13)

### DISCUSSION THREAD

**Discussion by filipesmedeiros** [vercel/next.js#41838] [Help]
On version 13, the default behaviour of the Link is to take children directly (instead of an `a` element). But in many cases we want to pass multiple children to a Link/a, for example to have text and an icon.

Does it make sense to have this limitation? https://nextjs.org/docs/messages/link-multiple-children

Seems that it just ported from the old behaviour (the default in Next12)

**Comment by icyJoseph** [comment]
Weird, that's behind a flag called `legacyBehavior` which in 13 is false. Are you perhaps changing it on your `next.config.js`?

I've tested and I don't get that error at all. 

```tsx
      <Link href="/" className="foo">
        <span>home</span>
        <i>h</i>
      </Link>
```

![Screenshot 2022-10-26 at 14 38 28](https://user-images.githubusercontent.com/21013447/198028049-abcc4ec1-e7ee-4500-b9fe-ab398f48301d.png)
